lib/uart.h

```diff
@@ -100,7 +109,14 @@ class Uart {
     uint8_t m_dlm;//Divisor Latch MSB
     uint8_t m_psd;//Prescaler Division
```

Comment:
I don't know if they need to be grouped in a struct since they are members of the class. Not dissimilar from what we're doing with the `Csr` class I think.

However it would be a good idea to init these to known values, or to something random by using the `irve_fuzzish_rand()` macro in `fuzzish.h` when register contents are unknown in hardware on reset.

Reply:
I think it's okay in `Csr` since the only members of that class are the regs themselves. Here, we have other members besides regs, so grouping them makes it clearer what they are for. Also, that will allow for things like random initializing the whole struct rather than having to do each reg individually

Reply:
Ah I see that makes sense. No strong preferences on my end :)